controls/cis_sle12.yml

```diff
@@ -1013,14 +1013,16 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    status: manual # rule is missing
+    status: manual
+    rules:
+    - set_iptables_outbound_n_established_connections_configured
```

Comment:
I would ask you to consider making this rule name shorter. It is related to  https://github.com/ComplianceAsCode/content/pull/10126#discussion_r1176250975